index.html

```diff
@@ -48,5 +48,157 @@ <h1 class="title">I’m Mariane<span id="info" class="title">Glad to see you!</s
+                        </div>
+                        <button type="button" class="viewProject">
+                            See Project
+                        </button>
+                    </div>
+                </li>
+            </ul>
+
+            <ul>
+                <li class="project">
+                    <img src="src/Snapshoot-Portfolio3.png" alt="Snapshot3">
+                    <div class="description">
+                        <h2 class="workstitle">Multi-Post Stories</h2>
+                        <div class="info">
+                            <p class="subTitle">Canopy</p>
+                            <ul class="infoContainer">
+                                <li class="role">Back End Dev</li>
+                                <li class="year">2015</li>
+                            </ul>
+                        </div>
+                        <p class="projectDescription">A daily selection of privately personalized reads;
+                             no accounts or sign-ups required.                  
+                        </p>
+                        <div class="tags">
+                            <ul class="tagsFlex">
+                                <li class="html">HTML</li>
+                                <li class="css">CSS</li>
+                                <li class="javascript">Javascript</li>
+                            </ul>
+                        </div>
+                        <button type="button" class="viewProject">
+                            See Project
+                        </button>
+                    </div>
+                </li>
+            </ul>
+    
+        </div>
+    </section>
+
+    <section class="about-section" id="About">
+        <h1 class="title">About<span class="title">Myself!</span>
+        </h1>
+
+        <p class="paragraph">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+
+        <p class="aboutConnect">LET’S CONNECT</p>
+    
+              <ul class="social-media-icons">
+                <li><a href=""><img class="icon twitter" 
+                    src="src/Twitter.png" alt="twitter-icon"/></a></li>
+
+                <li><a href=""><img class="icon linkedin" 
+                    src="src/Linkedin-icon.png" alt="linkedin-icon"/></a></li>
+
+                <li><a href=""><img class="icon msign" 
+                    src="src/m.png" alt="msign"/></a></li>
+
+                <li><a href=""><img class="icon github" 
+                    src="src/Github.png" alt="github-icon"/></a></li>
+
+                <li><a href=""><img class="icon finger" 
+                    src="src/Angel.png" alt="AngelList"/></a></li>
+              </ul>
+
+            <button type="button" class="resumeButton">Get my resume</button>
+               
+    </section>
```

Comment:
- [ ] Please make sure your design aligns with the original design guidelines(fonts, colors, margin, padding, etc)
- [ ] The button should look same as the original design and with same dimensions

### Your Design
![image](https://user-images.githubusercontent.com/43995830/153704181-87d6428f-177d-4850-a47a-15ede0b847cd.png)

### Original Design
![image](https://user-images.githubusercontent.com/43995830/153704195-7c94d003-1131-4f62-a413-48bfe782fe35.png)